Clinical-Genomics/cg
# change/q30-flowcell-check

## Description

Updated how Q30 threshold values for NovaSeq 6000 flow cells, according  AM dock 1830, are passed. In the future, if these values are to be changed, they can be done so through the new constants 

### Changed

- Q30 threshold values for samples

### How to prepare for test
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh add/q30-flowcell-check`

### How to test
- [x] do `cg transfer flowcell HMWTFDRXY `

### Expected test outcome

## Review
- [x] tests executed by @karlnyr 
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

## Implementation Plan
- [x] Document in *NA*
- [ ] Deploy this branch
- [x] Inform to @Clinical-Genomics/data-analysis
